### ISSUE TITLE
Problem with detecting changes when path contains non-ascii characters

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/qgis-mergin-plugin#227]
```
 Traceback (most recent call last):
              File "C:/Users/USERNAME/Roaming/QGIS/QGIS3\profiles\default/python/plugins\Mergin\plugin.py", line 227, in current_project_sync
              self.manager.sync_project(self.mergin_proj_dir)
              File "C:/Users/USERNAME/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\Mergin\projects_manager.py", line 183, in sync_project
              pull_changes, push_changes, push_changes_summary = self.mc.project_status(project_dir)
              File "C:/Users/USERNAME/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\Mergin\mergin\client.py", line 474, in project_status
              push_changes_summary = mp.get_list_of_push_changes(push_changes)
              File "C:/Users/USERNAME/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\Mergin\mergin\merginproject.py", line 344, in get_list_of_push_changes
              with open(result_file, 'r') as f:
             FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\USERNAME\\Google Drive (myemail@gmail.com)\\MY_MERGIN_PROJECT\\.mergin\\change_list1'
```

**Comment by wonder-sk** [comment]
Upstream geodiff issue https://github.com/lutraconsulting/geodiff/issues/113

**Comment by wonder-sk** [comment]
Upstream fix released in geodiff 0.8.7

**Comment by wonder-sk** [comment]
Fixed in mergin-py-client 0.5.11 and qgis-mergin-plugin 2021.2.1
